.github/workflows/flaky-tests-report.yml

```diff
@@ -0,0 +1,96 @@
+name: Flaky Tests Report
+
+on:
+  schedule:
+    # Run on Tuesdays and Fridays at noon Eastern time (5 PM UTC)
+    - cron: '0 17 * * 2,5'
+
+permissions:
+  contents: read
+  actions: read
+
+jobs:
+  flaky-tests-report:
+    runs-on: ubuntu-latest    
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.TEMPORAL_CICD_APP_ID }}
+          private-key: ${{ secrets.TEMPORAL_CICD_PRIVATE_KEY }}
+          owner: ${{ github.repository_owner }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
+        with:
+          version: ">=0.8.0"
+
+      - name: Install Python dependencies
+        run: |
+          cd tools/flakes
+          uv sync
+
+      - name: Install tringa
+        run: |
+          uv tool install git+https://github.com/dandavison/tringa@main --force
+
+      - name: Run tringa
+        id: generate-output-file
+        env:
+          GH_TOKEN: ${{ steps.generate_token.outputs.token }}
+        run: |
+          set -euo pipefail
+          
+          # Clean up any existing files
+          rm -rf out.json timeout.txt retry.txt flaky.txt crash.txt
+          
+          tringa --json --since-days 7 repo sql \
+            'select classname, name, count(*) as failure_count, max(artifact) as artifact from test where passed = false and skipped = false group by classname, name order by failure_count desc' \
+            --branch main \
+            --workflow-id 80591745 \
+            https://github.com/temporalio/temporal > out.json
+          
+          echo "✅ Tringa command completed"
+          echo "📊 Output file size: $(wc -c < out.json) bytes"
+          echo "📄 Full output file:"
+          cat out.json
+
+      - name: Run Python script to process flaky tests
+        id: process-flaky-tests
+        run: |
+          echo "Starting Python script processing..."
+          echo "Current directory: $(pwd)"
+          
+          echo "Running: cd tools/flakes && uv run main.py --file ../../out.json --github-summary --slack-webhook ${{ secrets.SLACK_WEBHOOK }} --run-id ${{ github.run_id }}"
```

Comment:
nit: I'd remove the echo details as they are likely going to be out-of-sync eventually.